docs/samples/Microsoft.ML.Samples.StaticPipe/Trainers.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Learners;
+using Microsoft.ML.StaticPipe;
+using System;
+
+namespace Microsoft.ML.Samples.StaticPipe
+{
+    public static class Trainers
+    { 
+        public static void SdcaRegression()
+        {
+            var (trainDataPath, testDataPath) = DatasetCreator.CreateRegressionDataset();
+
+            //creating the ML.Net IHostEnvironment object, needed for the pipeline
+            var env = new ConsoleEnvironment(seed: 0);
```

Comment:
(nit) do we want `ConsoleEnvironment` in the samples?  or `LocalEnvironment`?

Reply:
Personally I like the ConsoleEnvironmnet, so they see the ML.Net output during training. Is there an argument more in favor of LocalEnvironment?

---
In reply to: [222474325](https://github.com/dotnet/machinelearning/pull/1105#discussion_r222474325) [](ancestors = 222474325)

Reply:
I think the argument is that it isn't typical for libraries to write to the console. For example, if I use EntityFramework to read/write to a database, it doesn't by default print SQL statements to the console.

Also, it assumes that you are in a console app, when it is way more common to be in a UI app, or on an ASP.NET service.

Reply:
thanks, I'll change it. 

---
In reply to: [222493698](https://github.com/dotnet/machinelearning/pull/1105#discussion_r222493698) [](ancestors = 222493698)